flink-sql-connector-pulsar/pom.xml

```diff
@@ -90,11 +95,15 @@ under the License.
 							<relocations>
 								<relocation>
 									<pattern>org.bouncycastle</pattern>
-									<shadedPattern>org.apache.pulsar.shade.org.bouncycastle</shadedPattern>
+									<shadedPattern>org.apache.flink.connector.pulsar.shade.org.bouncycastle</shadedPattern>
 								</relocation>
 								<relocation>
 									<pattern>com.scurrilous</pattern>
-									<shadedPattern>org.apache.pulsar.shade.com.scurrilous</shadedPattern>
+									<shadedPattern>org.apache.flink.connector.pulsar.shade.com.scurrilous</shadedPattern>
+								</relocation>
+								<relocation>
+									<pattern>org.apache.pulsar.shade.io.netty</pattern>
+									<shadedPattern>org.apache.flink.connector.pulsar.shade.io.netty</shadedPattern>
```

Comment:
This shade is dangerous. I think. Make sure you know what you are doing now.

Reply:
Why? I'm trying to relocate all pulsar deps in the connector in the connector's pattern. This is related to https://github.com/apache/flink/pull/23003 also.

Is there any reason we use the same name for the connector shade class with the pulsar bundle one? It seems possibly cause `LinkageError`.